tsconfig.json

```diff
@@ -2,7 +2,11 @@
   "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
     "rootDir": ".",
-    "outDir": "build"
+    "outDir": "build",
+    "lib": [
+      "es2016",
+      "es2017.sharedmemory"
```

Comment:
@ofrobots PTAL at this in particular.  The `is` package we're using brings this in for the purposes of types, unless we pin to a really old version.  According to [node.green](https://node.green/#ES2017-features-shared-memory-and-atomics-SharedArrayBuffer) shared memory isn't really available until node 10, so this is a tough spot.  The options seem to be:
1. Include es2017.shared memory and be vigilant about not trying to use it (though CI will save us)
2. Just pin to an old version of `is` and don't update it for years (not likely)
3. Switch back to the other `is` package

Any thoughts?

Reply:
I vote for switching to `@sindresorhus/is`. It has more intuitive behavior – e.g. look at the difference between `is.error()` between the two, and it ships with types out of the box.

Reply:
Miscommunication on my part - we are currently using `@sindresorhus/is`, and it's the one with this type problem.